### PR TITLE
haxelib path fix

### PR DESCRIPTION
**Pull request by ProG4mr**

### CODE REVIEW

File: src/hxparse/Unexpected.hx [hunk 1]
```diff
@@ -20,6 +20,7 @@ class Unexpected<Token> extends ParserError {
 	public function new(token:Token, pos) {
 		super(pos);
 		this.token = token;
+		trace ('Unexpected $token at $pos');
```
Comment (Simn) [inline src/hxparse/Unexpected.hx]: Uhm... ;)

Comment (ProG4mr) [inline src/hxparse/Unexpected.hx]: Oops, must be something i left behind.

Comment (ProG4mr) [inline src/hxparse/Unexpected.hx]: Well spotted, think its done now.

**Comment by Simn** [comment]
Thanks!

**Pull request merged**